package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mics",
-  "version": "0.5.0",
+  "version": "0.6.1",
```

Comment:
Well, you might think about using [GitFlow](http://nvie.com/posts/a-successful-git-branching-model/), which is a lot more clear about that kind of thing. The version and final changes would be committed to a release branch, before landing in master, which always only contains a production-ready version.